streaming/streaming.go

```diff
@@ -361,6 +471,18 @@ func CleanUpStreamingJobs(ctx context.Context, conv *internal.Conv, projectID, r
 		return fmt.Errorf("datastream client can not be created: %v", err)
 	}
 	defer dsClient.Close()
+	pubsubClient, err := pubsub.NewClient(ctx, projectID)
+	if err != nil {
+		return fmt.Errorf("pubsub client cannot be created: %v", err)
+	}
+	defer pubsubClient.Close()
+	fmt.Println("Created pubsub client...")
```

Comment:
We should move `fmt.Println` statements to use the zap `Logger` instead. This is applicable at multiple places

Reply:
Done.